File: contoh.php/nilai2.php
[11.17, 27/10/2025] Ikhsan Mazon UNIDA: <?php
$a = readline("Masukan angk ke satu: ");
$b = readline("Masukan angka ke dua: ");

if($a > $b)
    echo "lebih terbesar dari: " . $a . "\n";
 elseif($b > $a)
    echo "lebih terbesar dari: " . $b . "\n";
 else 
    echo "Kedua sama besar: " . $a . "\n";

 echo "apakah keduanya sama?";
 echo ($a == $b) ? " Ya\n" : " Tidak\n";
?>
[11.17, 27/10/2025] Ikhsan Mazon UNIDA: a = int(input("Masukkan angka pertama: "))
b = int(input("Masukkan angka kedua: "))

if a > b:
    print(f"{a} lebih besar dari.") 
elif a < b:
    print(f"{b} lebih kecil dari.")    
else:
    print("Kedua angka sama besar.")

print("apakah keduanya sama ?", a == b)

File: contoh.cpp/nilai2.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b;
    cout << "Masukkan Angka Pertama: ";
    cin >> a;
    cout << "Masukkan Angka Kedua: ";
    cin >> b;

    if(a>b)
        cout << "lebih dari besar" << endl;
     else if(a<b)
        cout << "lebih dari kecil" << endl;
     else 
        cout << "Kedua Angka Sama Besar" << endl;

    cout<<"apakah kedua nya sama?"<< (a==b);
}